sceptre/helpers.py

```diff
@@ -62,3 +64,23 @@ def func_on_instance(key):
         for index, value in enumerate(attr):
             func_on_instance(index)
     return attr
+
+
+def normalise_path(path):
+    """
+    Converts backslashes to forward slashes.
+    Raises an PathConversionError if the path has a leading or
+    trailing slash.
+    :param path: A directory path
+    :type path: str
+    :raises: sceptre.exceptions.PathConversionError
+    :returns: A normalised path with forward slashes.
+    :returns: string
+    """
+    path = path.replace('\\', '/')
+    if path.endswith("/"):
```

Comment:
Just checking - why do we want to throw for trailing slashes? I think this could be quite annoying for people using `--dir` and tab completion to fill in the `project_path`.

Reply:
My thinking here is that we use `path.split()` through out the codebase and if there is a trailing slash I think split would return an empty string as the "head" rather than the last part of the path. E.g. 

```python
from os.path import split

with_slash = "this/is/a/path/" 
without_slash = "this/is/a/path"

split(with_slash) # returns ('this/is/a/path', '')
split(without_slash) # returns ('this/is/a', 'path')
```

I am not sure that sceptre has ever supported a trailing slash in stack/stackgroup names.